Extensions/AzureExp/Src/Tasks/ManageExperimentV1/task.json

```diff
@@ -40,24 +40,26 @@
             "required": true,
             "helpMarkDown": "Progression Id of the experiment to manage."
         },
-        {
-            "name": "ExperimentName",
-            "type": "string",
-            "label": "Experiment Name",
-            "required": true,
-            "helpMarkDown": "Name of the experiment to manage."
-        },
         {
             "name": "Action",
             "type": "pickList",
             "label": "Action",
             "required": true,
             "options": {
-                "Start": "Start the experiment",
-				"Advance": "Advance the experiment",
-                "Stop": "Stop the experiment"
+                "Start": "Start experiment",
+				        "Advance": "Advance experiment",
+                "Stop": "Stop experiment",
+                "StopAllExperiments": "Stop all experiments"
             },
             "helpMarkDown": "Select the action you want to perform"
+        },
+        {
+            "name": "ExperimentName",
+            "type": "string",
+            "label": "Experiment Name",
+            "required": true,
```

Comment:
It is not required for StopAll action.

Reply:
Visibility rule will handle the required scenario.